sympy/physics/mechanics/tests/test_linearize.py

```diff
@@ -263,6 +289,15 @@ def test_linearize_pendulum_lagrange_minimal():
     assert _simplify_matrix(A) == Matrix([[0, 1], [-9.8*cos(q1)/L, 0]])
     assert B == Matrix([])
 
+    # Check an alternative solver
+    if not USE_SYMENGINE:
+        # symengine doesn't support method='GJ'
```

Comment:
You can try as else `'FFLU'`, which is a fraction free LU solve only implemented in the symengine. There are by the way some fun solve methods, that might be worth looking into in the symengine.

Reply:
Interesting, didn't know that.